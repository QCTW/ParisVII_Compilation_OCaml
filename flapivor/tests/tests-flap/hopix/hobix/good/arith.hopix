fun succ (x) = x + 1

fun diff_sq (x, y) = x * x - y * y

val x = succ (diff_sq (10, 2))
