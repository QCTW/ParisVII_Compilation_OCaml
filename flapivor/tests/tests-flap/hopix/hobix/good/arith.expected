succ = <fun>
diff_sq = <fun>
x = 97
